lib/collection/src/collection/state_management.rs

```diff
@@ -205,7 +207,8 @@ impl Collection {
         }
 
         for (field_name, field_schema) in payload_index_schema.schema {
-            self.create_payload_index(field_name, field_schema).await?;
+            self.create_payload_index(field_name, field_schema, HwMeasurementAcc::disposable())
```

Comment:
would be nice to have a comment explaining why it is not tracked

Reply:
Added!